experimental/devices/mfrc522/mfrc522.go

```diff
@@ -0,0 +1,752 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// Package mfrc522 controls a Mifare RFID card reader.
+//
+// The code is largely ported from Python library : https://github.com/mxgxw/MFRC522-python
+//
+// The datasheet is available at : https://www.nxp.com/docs/en/data-sheet/MFRC522.pdf
```

Comment:
What I meant is to use the following form, so that then you can do something like https://periph.io/x/periph/experimental/devices/mfrc522#Datasheet but maybe it's just because I'm easily amused. :)

//
// Datasheet
//
// https://www.nxp.com/docs/en/data-sheet/MFRC522.pdf